webvtt_test.go

```diff
@@ -227,3 +227,9 @@ func TestWebVTTTags(t *testing.T) {
 Text with a <00:06:30.000>timestamp in the middle
 `, b.String())
 }
+
+func BenchmarkOpenWebVTT(b *testing.B) {
```

Comment:
Could you remove this benchmark and instead add a test to validate reading non-utf8 `webvtt` returns an error (in both header and afterwards)?